notebooks/218-vehicle-detection-and-recognition/218-vehicle-detection-and-recognition.ipynb

```diff
@@ -315,7 +315,7 @@
     "        if conf > threshold:\n",
     "            # Convert float to int and multiply corner position of each box by x and y ratio\n",
     "            # In case that bounding box is found at the top of the image, \n",
-    "            # we position upper box bar little bit lower to make it visible on image \n",
+    "            # you position upper box bar little bit lower to make it visible on image \n",
```

Comment:
l think it will bw better in general form, without pronounse, it is description how code works, so there is no actor